go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/klauspost/compress v1.15.15
 	github.com/klauspost/cpuid/v2 v2.2.4
+	github.com/mastercactapus/proxyprotocol v0.0.3
```

Comment:
They makes me think since we're pulling in this dependency, we should probably pull in https://github.com/mastercactapus/caddy2-proxyprotocol as a standard module as well (the listener wrapper).

Reply:
Sure, should I add it to modules/standard/imports.go?

Reply:
Standard modules need to be brought into this repo. We can ask @mastercactapus if he'd be alright with that.

Reply:
@mholt I'd be open to that

Reply:
I've gone ahead and pulled in the listener wrapper code into Caddy. Thanks so much @mastercactapus for having maintained it for all these years :blush: 

P.S. Maybe `github.com/mastercactapus/proxyprotocol` should transition to using `netaddr.IP`? It would be a breaking change but it might be slightly more efficient.